OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -423,13 +427,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <persName ref="PRS12986GabraKr"/>
                                 <seg type="ink">Black, red</seg>
                                 <date notBefore="1775" notAfter="1815"></date>
-                                <desc>
-                                    Fine handwriting
+                                <desc>Fine and regular handwriting, slightly deteriorating towards the end (e.g. <locus from="76v" to="78v"/>).
                                 </desc>
                                 <seg type="rubrication">
-                                    Holy names; several groups of lines on the incipit pages, alternating with black lines;
-                                    <!-- ... other elements ... -->
-                                    elements of the punctuation signs, text dividers, and numerals, included quire marks.
+                                    Holy names and the names of the individuals involved in the production of the book, both in the colophon and in the supplication formulas throughout the
+                                    manuscript. Several groups of lines on the incipit page of <ref target="#i2"/> and of the sections for each weekday, sometimes alternating with black lines. 
+                                    The words <foreign xml:lang="gez">ሰላም፡</foreign> in the colophon (<ref target="#i1"/>) and 
+                                    <foreign xml:lang="gez">ሰላም፡ ለኪ፡</foreign> in <ref target="#i4 #i5"/>. 
+                                    Elements of the punctuation signs, text dividers, and numerals, included quire marks.
```

Comment:
"and of the sections for each weekday," I would change to "and of the opening sections for each weekday"?

"Elements of the punctuation signs, text dividers, and numerals, included quire marks" "Elements of the punctuation, including text dividers, and numerals, including the quire marks."